Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -37,18 +48,13 @@ protected override IHost CreateHost(IHostBuilder builder)
             TestConfiguration.Create(b =>
             {
                 b.Sources.Clear();
-                b.AddConfiguration(configuration);
+                b.AddConfiguration(_configuration);
             });
         });
         builder.ConfigureContainer<ContainerBuilder>(ConfigureContainer);
         builder.UseSerilog((context, services, loggerConfiguration) =>
         {
-            // this allows serilog log statements to appear in the test console runner
-            // e.g. database sql queries (issued by DbContext) can be observed this way in the test
-            loggerConfiguration.WriteTo.Console(
-                restrictedToMinimumLevel: LogEventLevel.Warning,
-                formatProvider: CultureInfo.InvariantCulture,
-                outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] [{SourceContext}] {Message:lj}{NewLine}{Exception}");
+            loggerConfiguration.ConfigureSerilogForIntegrationTests();
```

Comment:
Move to shared extension class so that same settings can be used in the scheduler.